test/Microsoft.ML.Benchmarks/Helpers/BaseBenchClass.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using System;
+
+namespace Microsoft.ML.Benchmarks
+{
+   public class BaseBenchClass
+    {
+        public int _passed;
+
+        [IterationCleanup]
+        public void Dispose()
+        {
+            if (_passed < 0)
```

Comment:
FYI to any readers...

The negative return value from `Maml.MainCore()` is used due to AEther expecting a negative value:
https://github.com/dotnet/machinelearning/blob/d885ebc49c1fff04209714bb2757166a6f2681c9/src/Microsoft.ML.Maml/MAML.cs#L209-L210

Reply:
is AEther a tool ?

Reply:
Correct, it's a tool calling this code which drove the odd return values from `Maml.MainCore()`.